paint/preprocessing/binary_extractor.py

```diff
@@ -11,27 +11,27 @@
 
 class BinaryExtractor:
     """
-    Implement an extractor that extracts data from a binary file and saves it to h5 and json.
+    Implement an extractor that extracts preprocessing from a binary file and saves it to h5 and json.
 
-    This extractor considers data form a binary file that contains deflectometry data and heliostat properties. The data
-    is extracted and the deflectometry data saved in a h5 format and the heliostat properties as a json.
+    This extractor considers preprocessing form a binary file that contains deflectometry preprocessing and heliostat properties. The preprocessing
+    is extracted and the deflectometry preprocessing saved in a h5 format and the heliostat properties as a json.
 
     Attributes
     ----------
     input_path : Path
-        The file path to the binary data file that will be converted.
+        The file path to the binary preprocessing file that will be converted.
     output_path : Path
         The file path to save the converted h5 file.
     file_name : str
         The file name of the converted h5 file.
     raw_data : bool
-        Whether the raw data or filled data is extracted.
+        Whether the raw preprocessing or filled preprocessing is extracted.
     heliostat_id : str
         The heliostat ID of the heliostat considered in the binary file.
     json_handle : str
-        The file path to save the json containing the heliostat properties data.
+        The file path to save the json containing the heliostat properties preprocessing.
     deflectometry_created_at : str
```

Comment:
I could not make commit suggestions here (don't ask me why) but "preprocessing" should probably also be changed back to "data" here. I guess this comes from refactoring the former `data` subpackage to `preprocessing` in Pycharm?